.teamcity/settings.kts

```diff
@@ -0,0 +1,88 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.commitStatusPublisher
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    buildType(ChocolateyGUI)
+}
+
+object ChocolateyGUI : BuildType({
+    id = AbsoluteId("ChocolateyGUI")
+    name = "Build"
+
+    artifactRules = """
+        code_drop/MsBuild.log
+        code_drop/MSBuild.msi.log
+        code_drop/ChocolateyGUI.msi
+        code_drop/TestResults/issues-report.html
+        code_drop/Packages/**/*.nupkg
+    """.trimIndent()
+
+    params {
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.ChocolateyGUI_ChocolateyGuiVcsRoot%")
+        param("teamcity.git.fetchAllHeads", "true")
+        password("env.TRANSIFEX_API_TOKEN", "credentialsJSON:PUT_SOMETHING_HERE", display = ParameterDisplay.HIDDEN, readOnly = true)
+        password("env.GITHUB_PAT", "credentialsJSON:PUT_SOMETHING_HERE", display = ParameterDisplay.HIDDEN, readOnly = true)
```

Comment:
```suggestion
        password("env.GITHUB_PAT", "credentialsJSON:77847f9b-fb3c-59cb-be70-a708b3d3d1ed", display = ParameterDisplay.HIDDEN, readOnly = true)
```

I added the function to figure this out [here](https://gist.github.com/JPRuskin/8992e5665a3a73316bba3089de074078).